django/utils/autoreload.py

```diff
@@ -45,6 +46,16 @@
     pywatchman = None
 
 
+def is_django_module(module):
+    """Return True if the given module is nested under Django."""
+    return module.__name__.startswith('django.')
+
+
+def is_django_path(path):
+    """Return True if the given file path is nested under Django."""
+    return Path(django.__file__).parent in Path(path).parents
```

Comment:
This doesn't look to be used? (vs `django.utils.autoreload.is_django_path)

Reply:
I'm slightly confused: this is used in `django.template.autoreload`. I figured it makes more sense to define it here, but I can move it to the `django.template.autoreload` module if you'd prefer?

Reply:
I think I got confused myself here. Will review.